wiki/Tournaments/de.md

```diff
@@ -93,636 +93,636 @@ Nicht offizielle Turniere/Wettkämpfe, die von den Communities veranstaltet werd
+| [osu! Technical Championship #3](OTC/3) | 12.01.2019 | 02.03.2019 | ![][flag_IL] [Xilver15](https://osu.ppy.sh/users/3099689) | ![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050) | ![][flag_RO] [badeu](https://osu.ppy.sh/users/1473890) |
 
 #### osu! TV Size Tournament
 
 | Name | Start | Ende | ![Goldkrone][GCrown] | ![Silberkrone][SCrown] | ![Bronzekrone][BCrown] |
 | :-- | :-- | :-- | :-- | :-- | :-- |
-| osu! TV Size Tournament 2017 | 2017-03-04 | 2017-04-15 | European Dream | MudHorse | !^ |
-| [osu! TV Size Tournament 2018](otst!/2018) | 2018-02-17 | 2018-04-15 | Rubber Conquest | Floating up | black dogleg |
-| [osu! TV Size Tournament 2019](otst!/2019) | 2019-03-02 | 2019-04-13 | Floating up | Rubber Conquest | #arsecrackers |
-| [osu! TV Size Tournament 2020](otst!/2020) | 2020-02-29 | 2020-04-12 | Rubber Conquest | Floating up | seusbruk |
+| osu! TV Size Tournament 2017 | 04.03.2017 | 15.04.2017 | European Dream | MudHorse | !^ |
+| [osu! TV Size Tournament 2018](otst!/2018) | 17.02.2018 | 15.04.2018 | Rubber Conquest | Floating up | black dogleg |
+| [osu! TV Size Tournament 2019](otst!/2019) | 02.03.2019 | 13.04.2019 | Floating up | Rubber Conquest | #arsecrackers |
+| [osu! TV Size Tournament 2020](otst!/2020) | 29.02.2020 | 12.04.2020 | Rubber Conquest | Floating up | seusbruk |
 
 #### osu! UK Tournament
 
 | Name | Start | Ende | ![Goldkrone][GCrown] | ![Silberkrone][SCrown] | ![Bronzekrone][BCrown] |
 | :-- | :-- | :-- | :-- | :-- | :-- |
-| [UK osu!standard tournament!](OUKT/1) | 2013-11-11 | 2014-1-11 | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | ![][flag_GB] [R a h a r u](https://osu.ppy.sh/users/785193) | ![][flag_GB] [iLikeMudkipz](https://osu.ppy.sh/users/552515) |
-| [UK osu!standard tournament 2](OUKT/2) | 2014-03-15 | 2014-04-20 | ![][flag_GB] [iLikeMudkipz](https://osu.ppy.sh/users/552515) | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | ![][flag_GB] [SoraRoxas01](https://osu.ppy.sh/users/1986262) |
-| [osu! UK Tournament 3](OUKT/3) | 2014-06-28 | 2014-08-10 | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | ![][flag_GB] [jesus1412](https://osu.ppy.sh/users/230116) | ![][flag_GB] [iLikeMudkipz](https://osu.ppy.sh/users/552515) |
-| [osu! UK Tournament 4](OUKT/4) | 2015-01-10 | 2015-08-21 | ![][flag_GB] [bahamete](https://osu.ppy.sh/users/960620) | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | N/A |
-| [osu! UK Tournament 5](OUKT/5) | 2015-09-12 | 2016-04-23 | ![][flag_GB] [bahamete](https://osu.ppy.sh/users/960620) | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | ![][flag_GB] [Kakiage](https://osu.ppy.sh/users/3104031) |
+| [UK osu!standard tournament!](OUKT/1) | 11.11.2013 | 2014-1-11 | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | ![][flag_GB] [R a h a r u](https://osu.ppy.sh/users/785193) | ![][flag_GB] [iLikeMudkipz](https://osu.ppy.sh/users/552515) |
```

Comment:
```suggestion
| [UK osu!standard tournament!](OUKT/1) | 11.11.2013 | 11.01.2014 | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | ![][flag_GB] [R a h a r u](https://osu.ppy.sh/users/785193) | ![][flag_GB] [iLikeMudkipz](https://osu.ppy.sh/users/552515) |
```
The script didn't replace it because of the missing 0. This also needs to be adjusted in the English article.

Reply:
there were several more instances in english articles, but only this one in german articles. ty for catching